api/src/wfl/api/spec.clj

```diff
@@ -39,9 +39,28 @@
 
 (s/def ::workflow  (s/keys :req-un [::inputs]
                            :opt-un [::all/status ::all/updated ::all/uuid ::all/options]))
+
+;; This is the wrong thing to do. See [1] for more information.
+;; As a consequence, I've included the keys for a covid pipeline as optional
+;; inputs for batch workloads so that these keys are not removed during
+;; coercion.
+;; [1]: https://github.com/metosin/reitit/issues/494
+(s/def :batch/workload-request
```

Comment:
What motivated this move? It seemed to be in the right place before? Was there some dependency error?

Reply:
I attempted to move items, workload-inputs, inputs and workflow into the batch module, but in doing so, created a circular dependency as aou references batch in its implementation. Couldn't require spec in batch either as spec also requires batch.

Reply:
oh noes :'(

Reply:
Yes, the prior commit looked OK to me too FWIW.